drivers/include/xbee.h

```diff
@@ -104,6 +117,7 @@ typedef struct {
     gpio_t sleep_pin;                   /**< GPIO pin connected to SLEEP */
     ng_nettype_t proto;                 /**< protocol the interface speaks */
     uint8_t options;                    /**< options field */
+    uint8_t addr_flags;                 /**< address flags as defined above */
```

Comment:
why do you introduce a new flag? Just use one bit of the options flag (thats what it is for...).